pages/common/tmux.md

```diff
@@ -29,3 +29,7 @@
 - Kill session:
 
 `tmux kill-session -t {{name}}`
+
+- Kill session when attached:
+
+`Ctrl + B, x` (then hit 'y' for yes)
```

Comment:
Text isn't allowed outside of the backticks on the command examples.

Could you update this?